js/apps/system/_admin/aardvark/APP/frontend/js/views/collectionsView.js

```diff
@@ -498,7 +498,27 @@
           var tableContent = [];
           var advanced = {};
           var advancedTableContent = [];
-
+          var extendedNames = window.frontendConfig.extendedNames;
+          var traditionalNameValidation = [{
+            rule: Joi.string().regex(/^[a-zA-Z]/),
+            msg: 'Collection name must always start with a letter.'
+          },
+          {
+            rule: Joi.string().regex(/^[a-zA-Z0-9\-_]*$/),
+            msg: 'Only symbols, "_" and "-" are allowed.'
+          }];
+          var extendedValidation = [{
+            rule: Joi.string().regex(/^(?![0-9])/),
+            msg: 'Collection name cannot start with a number.'
+          },
```

Comment:
There is a rule missing here: collection names must not start with a `.` (dot), and, non-system collection names must not start with a `_` (underscore).

Reply:
Additionally, any control characters below `\u0020` are disallowed in collection names.

Reply:
@jsteemann should we also exclude `\u007F` (127 - DEL key)?